.vscode/.ropeproject/config.py

```diff
@@ -0,0 +1,85 @@
+# The default ``config.py``
+
+
+def set_prefs(prefs):
```

Comment:
Why did you check this in `.ropeproject/config.py ` ?

Reply:
Not sure how it came. removed it now